torba/torba/client/basedatabase.py

```diff
@@ -272,8 +272,8 @@ def _insert_sql(table: str, data: dict, ignore_duplicate: bool = False,
     def _update_sql(table: str, data: dict, where: str,
                     constraints: Union[list, tuple]) -> Tuple[str, list]:
         columns, values = [], []
-        for column, value in data.items():
-            columns.append("{} = ?".format(column))
+        for _, value in data.items():
```

Comment:
Sorry, that was a mistake. Reverting.

Reply:
@eukreign The names `column` and `columns` look too similar. Maybe we should change one name? Renaming `column` would be easier as its scope is local.

It would perhaps make the code more readable.

What do you think?